modules/fieldlinessequence/util/kameleonfieldlinehelper.cpp

```diff
@@ -104,7 +107,15 @@ bool convertCdfToFieldlinesState(FieldlinesState& state, const std::string& cdfP
     );
 
     state.setModel(fls::stringToModel(kameleon->getModelName()));
-    state.setTriggerTime(kameleonHelper::getTime(kameleon.get()));
+    double cdfDoubleTime = kameleonHelper::getTime(kameleon.get(), manualTimeOffset);
+    state.setTriggerTime(cdfDoubleTime);
+
+    //get time as string.
+    std::string cdfStringTime = SpiceManager::ref().dateFromEphemerisTime(cdfDoubleTime,
+        "YYYYMMDDHRMNSC::RND"); //YYYY MM DD HOUR MIN SEC   ROUNDED
+
+    // use time as string for picking seedpoints from seedm
+    std::vector<glm::vec3> seedPoints = seedMap.at(cdfStringTime);
 
     if (addLinesToState(kameleon.get(), seedPoints, tracingVar, state)) {
```

Comment:
Since `addLinesToState` is doing quite a bit of work, it would be better to instead:
```
bool success = addLinesToState
if (success) {
```
otherwise it's easy to miss the function when quickly browsing the code